### ISSUE TITLE
Switch Window

### ISSUE THREAD

**Issue by mtalreja** [adamdriscoll/selenium-powershell#17]
How can i switch window or tabs. I cannot see any functionalities provided for switching between them.

**Comment by mavericksevmont** [comment]
Supposing you have 2 tabs and $Driver = Start-SeChrome; try:

```
$Driver.SwitchTo().Window($Driver.WindowHandles[0])
$Driver.SwitchTo().Window($Driver.WindowHandles[1])

```

**Comment by mtalreja** [comment]
Thanks it worked 👍 
However i wanted to get some clarification. As when i looked into the Selenium Powershell Module i can't see the implementation of Switch and WindowHandles just was curious to know where these functionalities are provided.  

Thanks!!

**Comment by adamdriscoll** [comment]
Would be cool to make this into a couple cmdlets too. 

$Windows = Get-SeWindow -Driver $Driver
Switch-SeWindow -Driver $Driver -Window $Windows[0]
Switch-SeWindow -Driver $Driver -Window $Windows[1]

**Comment by the-mentor** [comment]
@adamdriscoll I've added initial Get-SeWindow/Switch-SeWindow cmdlets.
The only downside is that $Driver.WindowHandles doesnt return a readable string that is easy to understand.
Maybe in the future we can improve these cmdlets to be more friendly.

**Comment by adamdriscoll** [comment]
Resolved in #31
